src/main/java/edu/harvard/iq/dataverse/api/MakeDataCountApi.java

```diff
@@ -200,5 +206,51 @@ public Response updateCitationsForDataset(@PathParam("id") String id) throws IOE
             return wr.getResponse();
         }
     }
+    @GET
+    @Path("{yearMonth}/processingState")
+    public Response getProcessingState(@PathParam("yearMonth") String yearMonth) {
+        MakeDataCountProcessState mdcps;
+        try {
+            mdcps = makeDataCountProcessStateService.getMakeDataCountProcessState(yearMonth);
+        } catch (IllegalArgumentException e) {
+            return error(Status.BAD_REQUEST,e.getMessage());
+        }
+        if (mdcps != null) {
+            JsonObjectBuilder output = Json.createObjectBuilder();
+            output.add("yearMonth", mdcps.getYearMonth());
+            output.add("state", mdcps.getState().name());
+            output.add("state-change-timestamp", mdcps.getStateChangeTime().toString());
```

Comment:
I ran `mvn test -Dtest=MakeDataCountApiIT#testGetUpdateDeleteProcessingState` and the times seem to be stored in UTC, which is good. It's a little weird that we have a mix of camelCase and kebab-case:

```
{
    "status": "OK",
    "data": {
        "yearMonth": "2000-01",
        "state": "PROCESSING",
        "state-change-timestamp": "2024-03-27 18:02:07.41883"
    }
}
{
    "status": "OK",
    "data": {
        "yearMonth": "2000-01",
        "state": "DONE",
        "state-change-timestamp": "2024-03-27 18:02:08.012849829"
    }
}
```

Should we change this to `stateChangeTimestamp` for consistency?

Reply:
Fixed in b974f14 (thanks!)

Now it looks better:

```
{
    "status": "OK",
    "data": {
        "yearMonth": "2000-01",
        "state": "PROCESSING",
        "stateChangeTimestamp": "2024-03-27 18:59:07.979946"
    }
}
{
    "status": "OK",
    "data": {
        "yearMonth": "2000-01",
        "state": "DONE",
        "stateChangeTimestamp": "2024-03-27 18:59:08.58119825"
    }
}
```

Reply:
changed to stateChangeTimestamp